test/IDE/range_info_declattr.swift

```diff
@@ -7,11 +7,26 @@ class ObjCBase {
   }
   @objc var bar = 12, baz = 13
 }
+class Derived : ObjCBase {
+  @available(*, unavailable)
+  override var quux: Int {
+    @inlinable get { return 0 }
+  }
+
+  subscript(idx: Int) -> Int {
+    @available(*, unavailable)
+    get { return 0 }
+  }
```

Comment:
You should probably have a test case with multiple attributes on the same accessor, just in case. (The code looks correct for that already, but still.)

Reply:
Added!